plugins/pfring/Makefile.am

```diff
@@ -0,0 +1,8 @@
+pkglib_LTLIBRARIES = pfring.la
+
+pfring_la_SOURCES = runmode-pfring.c source-pfring.c plugin.c
+pfring_la_LDFLAGS = -module -avoid-version -shared
+pfring_la_LIBADD = -lpfring
+
+install-exec-hook:
+	cd $(DESTDIR)$(pkglibdir) && $(RM) $(pkglib_LTLIBRARIES)
```

Comment:
I should I add command that this is to remove the `.la` file automaker insists on installing but has no use for a plugin.